### DISCUSSION TITLE
Supporting @fromContext and @context directives

### DISCUSSION THREAD

**Discussion by awdng** [99designs/gqlgen#3297] [General]
Since May 2024 release of Federation v2.8 apollo router supports the new @fromContext and @context directives: https://www.apollographql.com/docs/federation/entities/use-contexts/
Since these are potentially interesting for us to use, I was wondering if there is already a plan to support this or if I am missing something and its already possible to use? 
Right now the codegen seems to fail and trying to just inject the directives still doesn't fully solve it but I still need to continuing my investigation on the topic.

Thanks in advance

**Comment by StevenACoffman** [comment]
It is not part of this:
https://www.apollographql.com/docs/federation/building-supergraphs/compatible-subgraphs/#go

But I'm happy to accept a PR to support it.
